pkg/codegen/generators/go_generator.go

```diff
@@ -182,8 +182,8 @@ import (
 	"strings"
 	"time"
 
-	"github.com/deepmap/oapi-codegen/pkg/runtime"
-	openapi_types "github.com/deepmap/oapi-codegen/pkg/types"
+	"github.com/oapi-codegen/runtime"
+	openapi_types "github.com/oapi-codegen/runtime/types"
```

Comment:
@spinillos not sure what's these are used for and/or how to test/verify. `make gen-cue` works at least 🤷

Reply:
If it works is enough :)

Reply:
💥